CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## v6.0.0-beta.6 - 2017-08-24
+
+### Engine updates and fixes
+
+* Make resource loading to work with PowerShell SxS installation (#4139)
+* Add missing assemblies to TPA list to make Pwrshplughin.dll work (#4502)
+* Make sure running `powershell` starts instance of the current version of PowerShell. (#4481)
+
+### General cmdlet updates and fixes
+
+* Fix Web CmdLets `-SkipHeaderValidation` to work with non-standard User-Agent headers. (#4479 & #4512) (Thanks @markekraus)
+* Use supported API to set Central Access Policy ID (CAPID) in SACL. (#4496)
+* Make `Start-Trace` support paths that require escaping in the underlying APIs (#3863)
+* Ignore case when binding PSReadline keyhandler functions (#4300) (Thanks @oising)
+* Removing `#if CORECLR` enabled, `Enable-PSRemoting` and `Disable-PSRemoting` (#4538)
```

Comment:
Is this the right PR #?  #4538 marks some feature tests as pending not for the cmdlets you listed.  Also, it seems that it's not worth mentioning per PR removal of `#if CORECLR` statements.